routes/alfresco.py

```diff
@@ -275,12 +275,7 @@ def run_fetch_alf_local_data(var_ep, lat, lon):
     if validation == 400:
         return render_template("400/bad_request.html"), 400
     if validation == 422:
-        return (
-            render_template(
-                "422/invalid_latlon.html", west_bbox=WEST_BBOX, east_bbox=EAST_BBOX
-            ),
-            422,
-        )
+        return render_template("422/invalid_latlon.html", west_bbox=WEST_BBOX, east_bbox=EAST_BBOX), 422
```

Comment:
My Python Black formatting converted this back to the original format that was changed here when saving this file locally. I think our Python Black configurations must be off. @charparr Are you seeing the same thing when you run the formatter on this file?

Reply:
@BobTorgerson same, here is my diff when I save that file (I `black` on save of every python file automatically in VS Code):
```
-        return render_template("422/invalid_latlon.html", west_bbox=WEST_BBOX, east_bbox=EAST_BBOX), 422
+        return (
+            render_template(
+                "422/invalid_latlon.html", west_bbox=WEST_BBOX, east_bbox=EAST_BBOX
+            ),
+            422,
+        )
```

The thing about Black is that there shouldn't even be a configuration: it is a strict, opinionated formatter. So long as we are running black on save or pre-commit then we shouldn't have any differences across our linting.
@Joshdpaul what is your setup?

Reply:
Ah yes, I've had this issue before with getting the remote connection to actually do the black format on save! Sometimes hard to tell when it's working or not.

I tried reinstalling the extension and re-saving the file, and line 278 now looks like your code above. Black adds parentheses around the return tuple, which is interesting...I originally edited that to be consistent with the 400 return line above, but if that's not necessary than we can go with the black formatting.

Reply:
Yeah, one pitfall I have found is that every time I connect to a new remote I need to re-install extensions. And so sometimes I can save a file, anticipating it will black, and then it isn't there. Two tell-tale black signals are that double quotes are nearly everywhere, rather than single quotes, and that all operators are bookended by white space, like `a=3` would be black formatted to `a = 3`.

Reply:
Yeah, it's no big deal! We all miss the formatting and it's honestly not a problem at all! Even if it slips through the cracks (which it most certainly does), it will be caught later by someone else and the correct formatting will be applied. This was an easy PR to identify that as being a potential formatting difference, so I wanted to ensure we weren't thrashing with config changes. I approve this change now!